{{cookiecutter.project_slug}}/manage.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+"""
+flask_script app manager
+"""
+import flask_script
+import {{ cookiecutter.project_slug }}.app
+
+
+def make_app_context():
+    return {'app': {{ cookiecutter.project_slug }}.app}
+
+
+#
+# Manage commands
+#
+# App manager
+#
+app = {{ cookiecutter.project_slug }}.app.create_app()
+manager = flask_script.Manager(app)
+
```

Comment:
Runserver is a default command, but I've found it helpful to set the default host by adding the following:

`manager.add_command('runserver', flask_script.Server(host='0.0.0.0'))`

Reply:
@imjohsep Why would I need to set the host? It's already in the Dockerfile.

Reply:
You could remove it from the Dockerfile if you set it here. Also, we don't exclusively run the applications in Docker.

Reply:
@imjohsep and I had a chat offline about this. Gonna leave the default behavior alone for now and re-visit if it becomes an issue.